script.embuary.info/resources/lib/helper.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
+# coding: utf-8
+
+########################
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+import json
+import time
+import datetime
+import os
+import operator
+import arrow
+import simplecache
+
+########################
+
+ADDON = xbmcaddon.Addon()
+ADDON_ID = ADDON.getAddonInfo('id')
+
+''' Python 2<->3 compatibility
+'''
+try:
+    ADDON_PATH = ADDON.getAddonInfo('path').decode('utf-8')
+except:
+    ADDON_PATH = ADDON.getAddonInfo('path')
+
+NOTICE = xbmc.LOGNOTICE
+WARNING = xbmc.LOGWARNING
+DEBUG = xbmc.LOGDEBUG
+
+DIALOG = xbmcgui.Dialog()
+
+CACHE = simplecache.SimpleCache()
+CACHE_ENABLED = ADDON.getSettingBool('cache_enabled')
+
+########################
+
+def log(txt,loglevel=NOTICE,force=False):
```

Comment:
Default log level should be debug (mandatory)

Reply:
changed